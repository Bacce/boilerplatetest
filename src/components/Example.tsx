import Styles from './Example.module.css';

export const Example = () => {
  return (
    <div className={Styles.container}>Example</div>
  );
}