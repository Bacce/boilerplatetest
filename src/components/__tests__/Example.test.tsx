import { render } from '@testing-library/react';
import { Example } from '../Example';

describe('Example Component', () => {    
    it('should render', ()=>{
        const view = render(<Example/>);
        
        expect(view.container).toMatchSnapshot();
    });
});
