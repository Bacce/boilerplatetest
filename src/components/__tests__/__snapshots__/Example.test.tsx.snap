// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Example Component should render 1`] = `
<div>
  <div
    class="container"
  >
    Example
  </div>
</div>
`;
